src/utils/useAxios/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { useState } from "react"
+import axios from 'axios';
+import { useToast } from "../../hooks/useToast";
+
+axios.defaults.baseURL = "";
+
+export const useAxios = () => {
+  
+  const operation = async (params:any) => {
```

Comment:
suggestion: `params` should be of type `AxiosRequestConfig`

Reply:
fixed. Thank for mentioning it.